bootstrap/migrate_redis_data.py

```diff
@@ -231,6 +231,7 @@ def process_done_batch(done_data: List[DoneData]):
     # Cache data
     cache_entries(data)
     save_incomplete_entries(data)
+    time.sleep(1)

```

Comment:
What is this for? I think this should either be removed or we should add a comment explaining why we need it

Reply:
it was to stop cloudflare from blocking requests and thinking that we were ddosing the site lol